chanzuckerberg/single-cell-data-portal
# fix: 6405 GE SVG download

## Reason for Change

- #6405

Followup - to prevent image download regression, we have #6381 in the pipeline to run visual tests

## Changes

1. To fix the SVG height not taking filtered cell types into account, resulting in the SVG having huge blank spaces `<SaveExport />` SVG download function is now using `allChartProps` like CSV download, in order to get the single source of truth on the tissues, genes, and expressions are rendered
2. Update SVG download to use the new constant for heat map cell height `HEAT_MAP_TOTAL_CELL_HEIGHT_PX` that was introduced 2 months ago
3. Update a fixture test URL to remove `compare` URL param, since it's not supposed to have it

SVG download now looks like this:

<img width="745" alt="Screenshot 2024-01-08 at 8 05 32 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/8085605b-6036-41e4-b529-055dc176fea3">

## Testing steps

1. Go to https://pr-6428-frontend.rdev.single-cell.czi.technology/gene-expression
2. Add some filters and genes
3. Download SVG
4. The downloaded file should be aligned now with no buggy huge white spaces like described in the ticket above

## Checklist 🛎️


## Notes for Reviewer
